master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -43,6 +43,7 @@ spec:
 | logSeveritySys | The log severity above which logs are sent to the syslog. Set to `NONE` for no logging to syslog. | DEBUG, INFO, WARNING, ERROR, CRITICAL, or NONE (case-insensitive) | string | `INFO` |
 | logSeverityFile| The log severity above which logs are sent to the log file. | Same as `logSeveritySys` | string | `INFO` |
 | logSeverityScreen | The log severity above which logs are sent to the stdout. | Same as LogSeveritySys | string | `INFO` |
+| ignoreLooseRPF\* | Allows Felix to run on systems with loose reverse path forwarding (RPF). | boolean | boolean | `false` |
```

Comment:
It might be nice if you could tell them whether `true` or `false` configures Felix to use loose reverse path forwarding. And also, I think it would be helpful to mention what Felix does by default.

Reply:
it's in the last column, which is the default value column

Reply:
So then, perhaps you could just change "Allows Felix to run on systems with loose reverse path forwarding (RPF)." to "Set to `true` to allow Felix to run on systems with loose reverse path forwarding (RPF)."